go/ql/test/library-tests/semmle/go/dataflow/ExternalFlowInheritance/mad_i2_subtypes_false.expected

```diff
@@ -0,0 +1,10 @@
+invalidModelRow
+paths
+| test.go:14:7:14:16 | call to Source | test.go:16:9:16:9 | y |
```

Comment:
Again noting expectations to cross-check: I think the embedding result is unexpected